ee/app/livechat-enterprise/server/hooks/afterRemoveDepartment.js

```diff
@@ -0,0 +1,9 @@
+import { callbacks } from '../../../../../app/callbacks';
+import { LivechatDepartment } from '../../../../../app/models/server';
+
+callbacks.add('livechat.afterRemoveDepartment', (department) => {
+	if (!department) {
+		return;
```

Comment:
We always need to return the same object we get through the callback.
```suggestion
		return department;
```